plugins/catalog-backend/src/service/createRouter.ts

```diff
@@ -228,6 +231,28 @@ export async function createRouter(
     });
   }
 
+  if (orchestrator) {
+    router.post('/validate-entity', async (req, res) => {
+      const bodySchema = z.object({
+        entity: z.unknown(),
```

Comment:
Let's call `validateEntityEnvelope` on the entity as well, just to make sure the structure is a bit sane. The orchestrator might throw some very confusion errors if it's not.